deploy/helm/distributed-compute-operator/values.yaml

```diff
@@ -38,6 +38,18 @@ istio:
   # Elevate pod execution permissions so that Istio's init container can modify
   # network settings when CNI plugin is NOT installed
   cniPluginInstalled: true
+  # namespace for istio mesh-level settings
+  rootConfigNamespace: istio-config
+  # set the HTTP connection idle timeout for pods managed by the distributed-compute-operator
+  httpIdleTimeout:
+    # the timeout to set. Leave empty for istio default.
+    timeout: ""
+    # If timeout and namespaces are set, sets the timeout only in given namespaces.
+    # If timeout is set but namespaces is empty:
+    #   - if config.watchNamespaces is empty, then timeout is set across the whole istio mesh
+    #   - else timeout is set for namespaces in config.watchNamespaces
+    # If any namespaces in the istio mesh are unaffected by this setting, they get istio default.
+    namespaces: []
```

Comment:
I don't think setting a timeout _across the whole istio mesh_ belongs here (i.e. this type of config belongs in the installer code). Creating a localized, targeted config is fine but not a cluster change.

I'm assuming `watchNamespaces` and `namespaces` are adequate, right?

Reply:
since the default `watchedNamespaces` is empty list which means "all namespaces", then the appropriate default for this http idle timeout is "across the whole istio mesh for pods managed by the DCO". that's what i was hoping the comment on line 43 communicated -- that its only for pods managed by the DCO

Reply:
what if i added "for pods managed by the DCO" to the end of that line 49 comment?